### DISCUSSION TITLE
Can query using DoH but DoT and DoQ time out

### DISCUSSION THREAD

**Discussion by sinkingfeeling** [AdguardTeam/AdGuardHome#5078] [Q&A]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not ask a question

### Operating system type

Linux, Other (please mention the version in the description)

### CPU architecture

AMD64

### Installation

Other (please mention in the description)

### Setup

On one machine

### AdGuard Home version

0.107.16

#### What did you do?

Tried to connect to query server via `DNS-over-TLS` and `DNS-over-QUIC` using Adguard VPN on Windows and iOS. My VPS has a domain name is has a SSL cert from Let's Encrypt.

#### Expected result

Expected to connect successfully.

#### Actual result

Connection timed out.

#### Screenshots (if applicable)

https://i.imgur.com/dGYKHWr.png

https://i.imgur.com/Sbqr2dK.png

#### Additional information

```
skarz@skarz-ubuntu:~$ dnslookup google.com tls://waguns.net
dnslookup v. 1.8.0-8187
2022/10/28 16:52:49 [fatal] Cannot make the DNS request: getting connection to tls://waguns.net:853: connecting to waguns.net: all dialers failed: 4 errors: "dial tcp 104.21.91.178:853: i/o timeout", "dial tcp 172.67.177.29:853: i/o timeout", "dial tcp [2606:4700:3031::6815:5bb2]:853: i/o timeout", "dial tcp [2606:4700:3032::ac43:b11d]:853: i/o timeout"
```

```
skarz@skarz-ubuntu:~$ dnslookup google.com https://waguns.net/dns-query
dnslookup v. 1.8.0-8187
dnslookup result:
;; opcode: QUERY, status: NOERROR, id: 25377
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version 0; flags: ; udp: 4096

;; QUESTION SECTION:
;google.com.	IN	 A

;; ANSWER SECTION:
google.com.	10	IN	A	142.251.215.238

```

**Comment by ameshkov** [comment]
This looks like only the port 443 is open on your VPS, all other ports are closed (iptables probably) and that's why you cannot access neither DoT nor DoQ.

**Reply by sinkingfeeling** [reply]
> This looks like only the port 443 is open on your VPS, all other ports are closed (iptables probably) and that's why you cannot access neither DoT nor DoQ.

Is this configured properly?

![image](https://user-images.githubusercontent.com/40373461/198850950-25ee5983-e233-41a5-b6b4-f994feb86ca4.png)

[edit] I updated my firewall via ufw, still receiving DNS_PPROBE_POSSIBLE.

![image](https://user-images.githubusercontent.com/40373461/198855385-1715c17b-2eb9-4740-934a-8ee06c04b9c4.png)

**Reply by ameshkov** [reply]
Yes, this looks right. You can also try running this command `telnet YOURIP 853`  to see if a TCP connection goes through.

**Reply by sinkingfeeling** [reply]
@ameshkov I feel silly now, the issue is on my end. It's because I am behind Cloudflare proxy which is prohibiting the connections via port 853.

**Reply by ameshkov** [reply]
No worries, it happens:)
